qa/L0_shared_memory/shared_memory_test.py

```diff
@@ -40,6 +40,8 @@
 import tritonclient.utils.shared_memory as shm
 from tritonclient import utils
 
+DEFAULT_SHM_BYTE_SIZE = 64
```

Comment:
nit: can we make `DEFAULT_SHM_BYTE_SIZE` a member of `SharedMemoryTest` class?

```
class SharedMemoryTest:
  def setUp(self):
    self.shared_memory_byte_size = 64
```

Reply:
Since it's a const, would it be ok to just do:
```
class SharedMemoryTest:
    DEFAULT_SHM_BYTE_SIZE=64
```
?

Reply:
I think it is fine as long as the variable does not change across instances of the class. I don't know about the all upper-case variable name, because the example from the Python docs tutorial use all lower-case.
https://docs.python.org/3/tutorial/classes.html#class-and-instance-variables
I will defer that to @Tabrizian .

Reply:
I was following: https://peps.python.org/pep-0008/#constants